plux/cli/cli.py

```diff
@@ -82,6 +83,7 @@ def main(argv=None):
     generate_parser = subparsers.add_parser(
         "entrypoints", help="Discover plugins and generate entry points"
     )
+    generate_parser.add_argument("-e", "--exclude", help="path(s) to exclude")
```

Comment:
a bit more explanation could make it easier to understand
```suggestion
    generate_parser.add_argument("-e", "--exclude", help="a sequence of paths to exclude; '*' can be used as a wildcard in the names. 'foo.*' will exclude all subpackages of 'foo' (but not 'foo' itself).")
```